File: client/src/app/shared/models/marketGroupDtl.ts

export interface IMarketGroupDtl {
    id: number;
    mstId: number;
    marketCode: string;
    marketName: string;
    sbu: string;
    sbuName: string;
    status: string; 
}
 
export class MarketGroupDtl implements IMarketGroupDtl {
    id: number;
    mstId: number;
    marketCode: string;
    marketName: string;
    sbu: string;
    sbuName: string;
    status: string; 
}

File: Core/Entities/MarketGroupDtl.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Core.Entities
{
    public class MarketGroupDtl:BaseEntity
    {
        public int MstId { get; set; }
        public string MarketCode { get; set; }
        public string  MarketName { get; set; }
        public string Status { get; set; }
    }
}
